admin_guide/manage_scc.adoc

```diff
@@ -260,6 +258,13 @@ For example, to allow the *e2e-user* access to the *privileged* SCC, run:
 $ oadm policy add-scc-to-user privileged e2e-user
 ----
 
+[WARNING]
+====
+Granting access to a user works only for the pods. For other resources
```

Comment:
Granting access to a user only works when the user directly creates a pod.  For pods created on behalf of a user, in most cases by the system itself, access ....

Examples of resources that create pods on behalf of a user are ...